src/Components/AllTvs.jsx

```diff
@@ -0,0 +1,42 @@
+//Deel 2 TechItEasy - Opdracht 2b, 2c
+
+import {inventory} from "../inventory.js";
+import {generateTvName} from "../helpers/generateTvName.js";
+import {formatPrice} from "../helpers/formatPrice.js";
+import {formatScreenSizes} from "../helpers/formatScreenSizes.js";
+import checkIcon from '../assets/check.png'
+import minusIcon from '../assets/minus.png'
+
+const AllTvs = () => {
+    return (
+        <div>
```

Comment:
Blijf letten op het gebruik van semantische elementen! Het zou netter zijn geweest om hier bijvoorbeeld een article van te maken en dan van de div in de volgende return een section.